src/main/java/mocacong/server/service/CafeService.java

```diff
@@ -156,15 +156,9 @@ public MyFavoriteCafesResponse findMyFavoriteCafes(Long memberId, Integer page,
     public MyReviewCafesResponse findMyReviewCafes(Long memberId, Integer page, int count) {
         Member member = memberRepository.findById(memberId)
                 .orElseThrow(NotFoundMemberException::new);
-        Slice<Cafe> myReviewCafes = cafeRepository.findByMyReviewCafes(memberId, PageRequest.of(page, count));
-        List<MyReviewCafeResponse> responses = myReviewCafes
-                .getContent()
-                .stream()
-                .map(cafe -> {
-                    int score = scoreRepository.findScoreByCafeIdAndMemberId(cafe.getId(), member.getId());
-                    return new MyReviewCafeResponse(cafe.getMapId(), cafe.getName(), cafe.getStudyType(), score);
-                })
-                .collect(Collectors.toList());
+
+        Slice<MyReviewCafeResponse> myReviewCafes = cafeRepository.findMyReviewCafesById(member.getId(), PageRequest.of(page, count));
```

Comment:
사소한 부분이지만 120자를 넘어가서 오른쪽으로 스크롤을 넘겨야 코드를 확인할 수 있네요.
개행을 추가해보면 어떨까요?
```suggestion
        Slice<MyReviewCafeResponse> myReviewCafes = 
        cafeRepository.findMyReviewCafesById(member.getId(), PageRequest.of(page, count));
```
<img width="1038" alt="image" src="https://github.com/mocacong/Mocacong-Backend/assets/57135043/a91b07d5-a773-438a-acc3-c0cd91dd2a5e">

Reply:
넵! 반영하고 푸쉬해놓을게요! 감사합니다! 🙂